src/matrix/cblas-wrappers.h

```diff
@@ -335,6 +336,9 @@ inline void cblas_Xsbmv1(
               1, x, 1, beta, y, 1);
 }
 
+inline void cblas_compile_time_assert (){
+	  KALDI_COMPILE_TIME_ASSERT (kNoTrans == CblasNoTrans && kTrans == CblasTrans);
```

Comment:
I'd rather you put this in a test program (e.g. matrix-lib-test.cc) to avoid polluting the header.

Reply:
Since we have C++11 now, you can simple do `static_assert(kNoTrans == CblasNoTrans && kTrans == CblasTrans, "informative error message")` in global scope in one of the .cc files. Probably easier to understand for the average C++ developer because it's part of the language.

Reply:
makes sense.  Our macro should be removed and replaced everywhere with that.

Reply:
put static_assert to the matrix-lib-test.cc. But we need make sure that numbers don't change from version to version and from matrix lib to another matrix lib.
from matrix-lib.h I simply removed include of cblas-wrappers.h - all compiles correctly, no need to add extra include to *.cc, header jama-svd.h also includes cblas-wrappers.h, but if it is not used outside it can not leaks symbols